src/Bynder/Api/Impl/OAuth2/RequestHandler.php

```diff
@@ -47,12 +47,13 @@ public function getAccessToken($code)
                 throw new \InvalidArgumentException('\'code\' cannot be empty or null when 
                 using authorization_code grant type.');
             }
-
+            echo "\n Using authorization_code flow";
```

Comment:
This will be rendered on websites using the SDK.

Reply:
yeah that's a fair point. I forgot this ain't python 😃